shared/src/metabase/mbql/schema.cljc

```diff
@@ -1501,6 +1501,8 @@
    (s/optional-key :dashboard-id) (s/maybe helpers/IntGreaterThanZero)
    (s/optional-key :pulse-id)     (s/maybe helpers/IntGreaterThanZero)
    (s/optional-key :nested?)      (s/maybe s/Bool)
+
+   (s/optional-key :metadata/dataset-metadata) (s/maybe [{s/Any s/Any}])
```

Comment:
It would be nice to add a line or two of comments explaining what the purpose of this is here. This namespace is as close to official documentation of MBQL as we currently have so it would be nice if we could try to make it a one-stop shop for wrapping your head around anything MBQL.

I know not everything here is adequately documented but I think the situation is slowly improving